media-video/droidcam/droidcam-1.4.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1 desktop linux-mod xdg-utils
+
+DESCRIPTION="Turn your mobile device into a webcam"
+HOMEPAGE="https://www.dev47apps.com/"
+SRC_URI="https://github.com/aramg/droidcam/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+RESTRICT="bindist mirror"
+
+DEPEND="gtk? (
+				dev-cpp/gtkmm:3.0
+				media-video/ffmpeg
+			)
+		!gtk? ( media-video/ffmpeg[-X] )
+		=app-pda/libusbmuxd-1*
+		media-libs/alsa-lib"
+RDEPEND="${DEPEND} dev-util/android-tools"
+BDEPEND="=media-libs/libjpeg-turbo-2*
+		>=media-libs/speex-1.2.0-r1
+		virtual/pkgconfig"
+
+S="${WORKDIR}/${P}/linux"
+DOCS=( README.md README-DKMS.md )
+DISABLE_AUTOFORMATTING="true"
+DOC_CONTENTS="
+		The default resolution for v4l2loopback-dc[1] is 640x480. You can override the
+		value by copying droidcam.conf.default to /etc/modprobe.d/droidcam.conf
+		and modifying 'width' and 'height'.
+
+		[1] https://github.com/aramg/droidcam/issues/56
+"
+
+BUILD_TARGETS="all"
+MODULE_NAMES="v4l2loopback-dc(video:${S}/v4l2loopback:${S}/v4l2loopback)"
+CONFIG_CHECK="~SND_ALOOP MEDIA_SUPPORT MEDIA_CAMERA_SUPPORT"
+ERROR_SND_ALOOP="CONFIG_SND_ALOOP: missing, required for audio support"
+
+PATCHES=(
+		"${FILESDIR}"/${P}-Makefile-fixes.patch
+)
+
+src_configure() {
+	true
+}
+
+src_prepare() {
+	default
+	if ! use gtk ; then
+		sed -i -e '/cflags gtk+/d' Makefile
+	fi
+	linux-mod_pkg_setup
+}
+
+src_compile() {
+	if use gtk ; then
+		emake droidcam
+	fi
+	emake droidcam-cli
+	linux-mod_src_compile
```

Comment:
I had prepended `KERNELRELEASE="${KV_FULL}" ` in my ebuild. This ensures that the module builds against the kernel selected by `eselect kernel` instead of the currently running kernel. (Useful when running `emerge @modules-rebuild` before rebooting to the new kernel)

Reply:
Sounds indeed useful, will put that in.